src/darwin/Framework/CHIP/MTRAttestationTrustStoreBridge.h

```diff
@@ -22,17 +22,23 @@ NS_ASSUME_NONNULL_BEGIN
 
 class MTRAttestationTrustStoreBridge : public chip::Credentials::AttestationTrustStore {
 public:
-    MTRAttestationTrustStoreBridge(NSArray<NSData *> * paaCerts)
+    MTRAttestationTrustStoreBridge(NSArray<NSData *> * paaCerts, NSArray<NSData *> * cdCerts)
         : mPaaCerts(paaCerts)
+        , mCDCerts(cdCerts)
     {
     }
     ~MTRAttestationTrustStoreBridge() {};
 
     CHIP_ERROR GetProductAttestationAuthorityCert(
         const chip::ByteSpan & skid, chip::MutableByteSpan & outPaaDerBuffer) const override;
 
+    CHIP_ERROR GetCertificationDeclarationCert(const chip::ByteSpan & skid, chip::MutableByteSpan & outCDDerBuffer) const override;
+
 private:
     NSArray<NSData *> * mPaaCerts;
+    NSArray<NSData *> * mCDCerts;
+
+    CHIP_ERROR GetCert(const chip::ByteSpan & skid, chip::MutableByteSpan & outDerBuffer, NSArray<NSData *> * certs) const;
```

Comment:
We have to implement in Linux chip-tool as well

Reply:
Looks resolved

Reply:
Comment tracked in #22439